test/extended/cni_vendor_test.sh

```diff
@@ -2,10 +2,15 @@
 
 source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
 
-# Set this to false if namespaces are not isolated
-export OPENSHIFT_NETWORK_ISOLATION="${OPENSHIFT_NETWORK_ISOLATION:-true}"
+# Set this to false if the plugin does not implement NetworkPolicy
+export NETWORKING_E2E_NETWORKPOLICY="${NETWORKING_E2E_NETWORKPOLICY:-true}"
+
+# Set this to true if the plugin implements isolation in the same manner as
+# redhat/openshift-ovs-multitenant
+export NETWORKING_E2E_ISOLATION="${NETWORKING_E2E_ISOLATION:-false}"
```

Comment:
Ack. Good for now. I wasn't sure because network policy is still experimental/beta. And openshift is still locked on assuming project level isolation.